microsoft-edge/webview2/reference/javascript/webvieweventmap.yml

```diff
@@ -0,0 +1,35 @@
+### YamlMime:TSType
+name: WebViewEventMap
+uid: 'WebView2Script!WebViewEventMap:class'
+package: WebView2Script!
+fullName: WebViewEventMap
+summary: ''

```

Comment:
Need to include a summary, which is also displayed on the top level page in the JS ref docs.

https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/reference/javascript/?branch=pr-en-us-2315